### PR TITLE
Add tutorial for auto exporting to S3 in TypeScript

### PR DESCRIPTION
**Pull request by melchii**
*Issue #, if available:*

*Description of changes:*
Adds an example for how to create the infrastructure for auto exporting to S3 as a subscriber using the AWS CDK along with the lambda code for exporting.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: subscribers/javascript/auto-export-to-s3/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+$ cdk bootstrap (only needed if this is your first time running cdk)
+
+$ npm run-script deploy
+```
+
+The CloudFormation will create the following resources:
+
+1. S3 Bucket
+1. Lambda function (with necessary IAM permissions)
+1. CloudWatch Rule
```
Comment (dblock) [inline subscribers/javascript/auto-export-to-s3/README.md]: function lowercase but Rule, decide :)

File: subscribers/javascript/auto-export-to-s3/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Each time a new Revision is added to a Data Set to which your account is subscribed, the Assets will be automatically exported to S3 as a response to the CloudWatch Event sent by AWS Data Exchange.
```
Comment (dblock) [inline subscribers/javascript/auto-export-to-s3/README.md]: DataSet

File: subscribers/javascript/auto-export-to-s3/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "auto-export-to-s3",
+  "version": "1.0.0",
+  "description": "Auto export new Junto Assets to an Amazon S3 Bucket.",
```
Comment (dblock) [inline subscribers/javascript/auto-export-to-s3/package.json]: Junto -> AWS Data Exchange

File: subscribers/javascript/auto-export-to-s3/package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws-samples/aws-dataexchange-api-samples.git"
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "ISC",
```
Comment (dblock) [inline subscribers/javascript/auto-export-to-s3/package.json]: This should be MIT as the repo. I think you should also put your name above.

File: subscribers/javascript/auto-export-to-s3/package.json (hunk 3 above)
Comment (melchii) [reply subscribers/javascript/auto-export-to-s3/package.json]: Done

File: subscribers/javascript/auto-export-to-s3/package.json (hunk 4 above)
Comment (melchii) [reply subscribers/javascript/auto-export-to-s3/package.json]: Done

File: subscribers/javascript/auto-export-to-s3/README.md (hunk 2 above)
Comment (melchii) [reply subscribers/javascript/auto-export-to-s3/README.md]: Done

File: subscribers/javascript/auto-export-to-s3/README.md (hunk 1 above)
Comment (melchii) [reply subscribers/javascript/auto-export-to-s3/README.md]: Done

File: subscribers/javascript/auto-export-to-s3/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Auto Export To S3 (JavaScript)
+
+This sample shows how to set up an AWS Lambda function which will automatically export all newly published Revisions to S3. All infrastructure is setup using the AWS CDK.
```
[the remark this answers is not in the document]
Comment (melchii) [reply subscribers/javascript/auto-export-to-s3/README.md]: Done

**Comment by melchii** [comment]
Updated!

**Pull request merged**